sdk/AzSdk.reference.props

```diff
@@ -0,0 +1,36 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.props'))" />
```

Comment:
Why are these reference.props files needed? Should the one under eng be getting picked up?

Reply:
The ones under eng are `reference.targets`. Either way the build broke without bthem so I copied them from the old src path